apps/zui/src/js/models/lake.ts

```diff
@@ -1,23 +0,0 @@
-import {Lake} from "../state/Lakes/types"
-
-export type LakeModel = ReturnType<typeof lake>
-
-export default function lake(lake: Lake) {
-  return {
-    ...lake,
-    getAddress(): string {
-      return this.port ? [this.host, this.port].join(":") : this.host
-    },
-    serialize(): Lake {
-      return {
-        host: this.host,
-        id: this.id,
-        name: this.name,
-        port: this.port,
-        version: this.version,
-        authType: this.authType,
-        authData: this.authData,
-      }
-    },
-  }
-}
```

Comment:
This is what the lake.ts model used to look like. No it looks like the code in the next comment.